boxoffice/views/item_collection.py

```diff
@@ -13,7 +13,7 @@ def jsonify_item(item):
             'title': item.title,
             'id': item.id,
             'description': item.description.text,
-            'quantity_available': item.quantity_available,
+            'quantity_available': item.quantity_total - item.get_confirmed_line_items.count(),
```

Comment:
Why is `item.quantity_available` unreliable here, and why is that column present if it is not reliable?